### PR TITLE
Add background prop to background util

### PR DESCRIPTION
**Pull request by MrTibbles**
- Adds the shorthand CSS value `background` to the background util styles object.

### CODE REVIEW

File: lib/Image/Image.js [hunk 1]
```diff
@@ -13,7 +13,10 @@ const Image = styled.div`
   ${border};
   ${boxModel};
   ${css};
-  background-image: url(${({ src }) => src});
+  ${({ src }) =>
```
Comment (lukehedger) [inline lib/Image/Image.js]: I wonder if we should remove this `src` attribute altogether as it isn't really inline with the rest of the library and is a bit misleading as it is not actually an ``.

So, instead you would just do:
```jsx
<Image backgroundImage="url('...')" />
```

Also, I guess this is not strictly related to this PR. I'm happy with the other changes but maybe this point is a wider discussion around the Image primitive?

Comment (MrTibbles) [reply lib/Image/Image.js]: Yeah, i agree, it is misleading as to what DOM element is produced. But as you say, that should be picked up in a separate PR.

Comment (lukehedger) [reply lib/Image/Image.js]: Maybe revert this change then?

Comment (MrTibbles) [reply lib/Image/Image.js]: This is a fix. Changing `src` to `backgroundImage` would be a breaking change.

Comment (lukehedger) [reply lib/Image/Image.js]: I mean just leave it as `background-image: url(${({ src }) => src});`

Comment (MrTibbles) [reply lib/Image/Image.js]: Right, now i get your point. Yeah it is out of scope of this PR, but we did actually agree to include it when initially creating this PR. Lets discuss tomorrow.
